src/components/screens/dashboard/components/ChangeAuthView.tsx

```diff
@@ -84,7 +78,7 @@ const ChangeAuthView: React.FC = () => {
             try {
               await setPreferredAuthenticatorSdk(authenticator, setMessage);
             } catch (err) {
-              handleError(err);
+              logoutOnInvalidToken(err);
             }
             await updateAuthenticators();
```

Comment:
should updateAuthenticators still be called after a user is logged out

Reply:
No, but let's not overcomplicate it. It will work just fine but just try to get the authenticators afterwards which will silently fail.